src/templates/page.html.template

```diff
@@ -2,10 +2,22 @@
 <html>
 <head>
     <meta charset="utf-8">
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="stylesheet" href="/${css_dir}/normalize.css" type="text/css" media="all">
-    <link rel="stylesheet" href="/${css_dir}/style.css" type="text/css" media="all">
+    <meta name="description" content="${page_title}">
+
+    <link rel="manifest" href="/site.webmanifest">
+
+    <link rel="apple-touch-icon" sizes="180x180" href="/${assets_dir}/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="/${assets_dir}/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="/${assets_dir}/favicon-16x16.png">
+    <link rel="favicon" href="/favicon.ico"/>
```

Comment:
Yeah, I thought of something like this too, specially with JS files as the only JS for now is the template one.

I think it would be nice to have the `meta\` folder, as it has a very clear purpose.

I didn't make this change on this PR because I tried to make as little change as I could but since you brought it up, I believe now is the time!

Reply:
Yea, I agree. The JS file is only a template currently, but that might chance in the future, if additional JS files will be added, that don't require templating, e.g. for #25.

A `meta` directory seems the way to go!

Reply:
I made the changes to add the `meta` folder. @766F6964 let me know what you think.

Reply:
Looks good to me! However, I think it would be sensible to also document the purpose of the `meta` directory in the README, under the "Structure Explanation" table.

Reply:
I improved the documentation and I did some changes to the URL generations, as well.

*nix is always "/" which is the same for URLs but Windows uses "\" so some URLs were being generated with the Windows bar...

Let me know if there is anything else.